### DISCUSSION TITLE
my options.lua file is being overridden for some reason...

### DISCUSSION THREAD

**Discussion by Ochiruko** [LazyVim/LazyVim#5134] [Q&A]
I installed lazyvim from the starter a couple of hours ago, and after reading the docs, it seemed like generic options should be placed in options.lua. So off I went, to add `vim.opt.spell = false`. For some reason though, it wasn't working, so I spent, like, 2 hours scouring the web for why (my internet skills are admittedly weak). I've tried adding other settings, like `vim.opt.relativenumber = false`, and they actually worked. I suspect, therefore, that some other, more specific file (autocmd.lua?) is overriding it. There's nothing in autocmd.lua that turns on spellcheck, however, and I'd like to turn it off globally anyways. What should I do here?

**Comment by yusufaktepe** [comment]
Overridden by [this autocmd](https://github.com/LazyVim/LazyVim/blob/2d7a04be47b94010fd6c1a2be0283c0d4e5738ae/lua/lazyvim/config/autocmds.lua#L97).

Disable it with:
```lua
vim.api.nvim_del_augroup_by_name("lazyvim_wrap_spell")
```

**Reply by agriffis** [reply]
Just to add to what @yusufaktepe said, this is the recommended approach as shown here:

https://www.lazyvim.org/configuration/general#auto-commands

**Comment by Ochiruko** [comment]
Thanks, I missed that!
